### DISCUSSION TITLE
HTTPUpgrade behind Nginx

### DISCUSSION THREAD

**Discussion by bart3nder** [v2fly/v2ray-core#2843] [Q&A]
Hi! How can I use HTTPUpgrade transport behind Nginx? what headers should I set?

**Comment by iopq** [comment]
https://github.com/v2fly/v2ray-core/pull/2727 doesn't give nginx config so it's not working for me

**Reply by athrun-dev** [reply]
For HTTPUpgrade I tried the same configuration as WebSocket and it works fine.
Maybe you can refer to this [repository](https://github.com/lxhao61/integrated-examples) for more templates. Caddy configurations also included.
